Core/Page Object/Base.ts

```diff
@@ -0,0 +1,22 @@
+import {Locator, Page} from '@playwright/test';
+import {Dashboards} from "./dashbord";
+import {LoginPage} from "./loginPage";
+
+export class BasePage {
+  readonly page: Page;
+  readonly dashboardPage: Dashboards;
+  readonly loginPage: LoginPage;
+  readonly sideBarMenu:Locator
+
+  constructor(page: Page) {
+    this.page = page
+    this.loginPage = new LoginPage(this.page);
```

Comment:
It is a bad idea to create these instances here. It's not in the logic of BasePage at all, also there is no guarantee that you'd need all the pages in your code, though you create all of them at once. Imagine that you'd have 100 pages, but will need only 1, how would you handle that? I'd suggest to create index.ts file, where you'd have method which would initialize and return needed page based on the argument that you pass for it, like const loginPage = getPage('login'). And getPage() method will create and return needed instance